portfolio/pom.xml

```diff
@@ -47,7 +47,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>ethanimooney-step-2020</deploy.projectId>
+          <deploy.projectId>ethanimooney-step-apis</deploy.projectId>
```

Comment:
Did this work for you locally? No need to change it if it doesn't bring value.

Reply:
It works, just need billing to be setup

Reply:
Gotcha.